cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/executor/Executor.java

```diff
@@ -499,33 +485,31 @@ private boolean isLeadershipMovementDone(Cluster cluster, TopicPartition tp, Exe
 
     }
 
-    private boolean maybeMarkTaskAsDeadOrAborted(Cluster cluster, ExecutionTask task, boolean rollbackOnStopping) {
+    /**
+     * Mark the task as aborting or dead if needed.
+     *
+     * Ideally, the task should be marked as:
+     * 1. ABORTING: when the execution is stopped by the users.
+     * 2. ABORTING: When the destination broker is dead so the task cannot make progress, but the source broker is
+     *              still alive.
+     * 3. DEAD: when both the source and destination brokers are dead.
+     *
+     * Currently KafkaController does not support updates on the partitions that is being reassigned. (KAFKA-6034)
+     * Therefore once a proposals is written to ZK, we cannot revoke it. So the actual behavior we are using is to
```

Comment:
To avoid long time incurred by execution cancellation due to KAFKA-6034, should we set `num.concurrent.partition.movements.per.broker` to `1` in `cruisecontrol.properties`?